Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -522,6 +522,7 @@ then
     else echo "Unsupported/unknown product: ${PRODUCT_NAME}"
     fi
 elif [[ "${WK_PLATFORM_NAME}" == iphoneos ||
+        "${WK_PLATFORM_NAME}" == iphonesimulator ||
```

Comment:
What does this do? Given the `if [[ "${WK_PLATFORM_NAME}" =~ .*simulator ]]` branch above, I would think that this condition would never be evaluated when we're building for `iphonesimulator`.

Reply:
Turns out this bit of code for sim support should exist elsewhere. Removed.